### ISSUE TITLE
Brew reinstall fails 

### ISSUE THREAD

**Issue by timlinux** [qgis/homebrew-qgisdev#26]
Hi @dakcarto 

Testing with your updates from today, I can't brew reinstall as suggested:

```
[timlinux@sandstone ~/dev/cpp/QGIS-CLion-Build]$ brew install --only-dependencies qgis3-dev
==> Installing qgis3-dev from qgis/qgisdev
qgis3-dev: Unsatisfied dependency: httplib2
Homebrew does not provide special Python3 dependencies; install with:
  `pip3 install httplib2`

You may need to: `brew install python3`

qgis3-dev: Unsatisfied dependency: pygments
Homebrew does not provide special Python3 dependencies; install with:
  `pip3 install pygments`

You may need to: `brew install python3`

qgis3-dev: Unsatisfied dependency: jinja2
Homebrew does not provide special Python3 dependencies; install with:
  `pip3 install jinja2`

You may need to: `brew install python3`

qgis3-dev: Unsatisfied dependency: owslib
Homebrew does not provide special Python3 dependencies; install with:
  `pip3 install owslib`

You may need to: `brew install python3`

Error: Unsatisfied requirements failed this build.
```

Note that python3 is installed:

```
[timlinux@sandstone ~/dev/cpp/QGIS-CLion-Build]$ brew list | grep python3
python3

```

See also #22

**Comment by timlinux** [comment]
Ah wait maybe it is just me needing to install those pip packages too

**Comment by timlinux** [comment]
Confirmed....closing

**Comment by dakcarto** [comment]
Hi @timlinux,

Yeah, @jef-n recently [pulled many of those packages](https://github.com/qgis/QGIS/commit/f10e9eee7db8eb00100007955c3b92a27d45bf52) from the source tree, so now they need to be dependencies for the formula. Maybe we need to add `runtime-requirements.txt` (`psycopg2`, etc) and `dev-requirements.txt` (`mock`, etc.) to the [`src/python/ext-libs/` directory](https://github.com/qgis/QGIS/tree/master/python/ext-libs)?

Then, if any are missing, just recommend the user do `pip3 install -r [dev-|runtime-]requirements.txt `?

**Comment by timlinux** [comment]
> Then, if any are missing, just recommend the user do pip3 install -r [dev-|runtime-]requirements.txt?
@dakcarto Yeah that sounds good!

**Comment by dakcarto** [comment]
Reopening, as this needs implemented.

**Comment by dakcarto** [comment]
Closing, as fixed in 2f972fe. Reopen if necessary.
